### PR TITLE
Fix linting errors.

### PR DESCRIPTION
**Pull request by rxl881**

### CODE REVIEW

**Comment by ara4n** [comment]
I **HATE** our linter. thanks.

File: src/Tinter.js [hunk 1]
```diff
@@ -376,8 +383,7 @@ class Tinter {
                     const attr = this.svgAttrs[k];
                     for (let l = 0; l < this.keyHex.length; l++) {
                         if (tag.getAttribute(attr) &&
-                            tag.getAttribute(attr).toUpperCase() === this.keyHex[l]) 
-                        {
+                            tag.getAttribute(attr).toUpperCase() === this.keyHex[l]) {
```
Comment (lukebarnard1) [inline src/Tinter.js]: the style we would generally go with is
```
if (reallyLongstuff &&
    otherReallyLongStuff
) {
    bla

Comment (ara4n) [reply src/Tinter.js]: says who, though? i've never used that style in my life, and it's not in our code style (https://github.com/matrix-org/matrix-react-sdk/blob/master/code_style.md), and I don't understand why we are enforcing lint rules which are more extreme than our code style!

Comment (lukebarnard1) [reply src/Tinter.js]: I've never seen that, actually. We really ought to update the linting configuration to reflect that.

Comment (lukebarnard1) [reply src/Tinter.js]: I don't find it that bad, but then I do have the linting built into my IDE, so there's no extra "appeasing" step in general.

**Pull request merged**